src/V3EmitCSyms.cpp

```diff
@@ -440,18 +444,27 @@ void EmitCSyms::emitSymHdr() {
     }
     puts("bool __Vm_didInit;\n");
 
-    puts("\n// SUBCELL STATE\n");
+    if (v3Global.opt.mtasks()) {
+        puts("VlThreadPool* const __Vm_threadPoolp;\n");
+        puts("bool __Vm_even_cycle;\n");
+
+        if (v3Global.opt.profThreads()) {
+            // rdtsc() at current cycle start
+            puts("vluint64_t __Vm_profile_cycle_start;\n");
+            // Time we finished analysis
+            puts("vluint64_t __Vm_profile_time_finished;\n");
+            // Track our position in the cache warmup and actual profile window
+            puts("vluint32_t __Vm_profile_window_ct;\n");
```

Comment:
```suggestion
            puts("vluint64_t __Vm_profile_cycle_start = 0;\n");
            // Time we finished analysis
            puts("vluint64_t __Vm_profile_time_finished = 0;\n");
            // Track our position in the cache warmup and actual profile window
            puts("vluint32_t __Vm_profile_window_ct = 0;\n");
```

My earlier mess - I pushed to trunk fixing this.